### PR TITLE
Separate tasks

### PR DESCRIPTION
**Pull request by pepe**
Separate cljs-devtools and dirac tasks as discussed in #3

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -1,32 +1,57 @@
...
 Latest version:
 
 [[https://clojars.org/powerlaces/boot-cljs-devtools][https://img.shields.io/clojars/v/powerlaces/boot-cljs-devtools.svg]]
 
 In order to install it, add the following to your =build.boot= dependencies:
 #+BEGIN_SRC clojure
-[binaryage/devtools      "0.8.2" :scope "test"]
-[binaryage/dirac         "0.7.1" :scope "test"]
+[binaryage/devtools      "0.8.3" :scope "test"] ;; when you want cljd-devtools or both
+[binaryage/dirac         "0.8.7" :scope "test"] ;; when you want dirac or both
```
Comment (arichiardi) [inline README.org]: Maybe this one should definitely be X.X.X or `RELEASE` or we'll keep updating it every time

File: src/powerlaces/boot_cljs_devtools.clj [hunk 2]
```diff
@@ -9,9 +9,8 @@
 
 (def ^:private deps '#{binaryage/devtools binaryage/dirac})
 
-(defn- add-preloads! [in-file out-file]
-  (let [preloads ['devtools.preload 'dirac.runtime.preload]
-        spec (-> in-file slurp read-string)]
+(defn- add-preloads! [preloads in-file out-file]
```
Comment (arichiardi) [inline src/powerlaces/boot_cljs_devtools.clj]: Can easily become a multimethod I guess, where we dispatch on the preloads type. Surely overkill :smile: feel free to ignore me if you don't have time, I'll merge it anyways.

File: src/powerlaces/boot_cljs_devtools.clj [hunk 3]
```diff
@@ -52,16 +57,34 @@
...
   [b ids        BUILD_IDS  #{str} "Only inject devtools into these builds (= .cljs.edn files)"
    n nrepl-opts NREPL_OPTS edn     "Options passed to boot's `repl` task."
    d dirac-opts DIRAC_OPTS edn     "Options passed to dirac."]
-  (let [tmp (boot/tmp-dir!)
-        prev (atom nil)
-        nrepl-opts (cond-> (merge nrepl-defaults nrepl-opts)
-                     (get-in dirac-opts [:nrepl-server :port]) (assoc :port (get-in dirac-opts [:nrepl-server :port])))
-        dirac-opts (cond-> (or dirac-opts {})
-                     (:port nrepl-opts) (assoc-in [:nrepl-server :port] (:port nrepl-opts)))
+  (let [tmp              (boot/tmp-dir!)
```
Comment (arichiardi) [inline src/powerlaces/boot_cljs_devtools.clj]: Can I ask you to clean this spacing issue? Too much green in between parens, one space is good enough, again, I'll merge it anyways so if you don't have time not a big deal.

Comment (arichiardi) [reply src/powerlaces/boot_cljs_devtools.clj]: Ok these guys are aligned so maybe I just need to know how to make it so :smile: I use `aggressive-indent-mode` which just keeps everything formatted following the clojure style guide and probably does not like this :crying_cat_face:

**Review by arichiardi — COMMENTED** [review]
Looks good to me, minor changes, but as I said I will merge it anyways.

File: src/powerlaces/boot_cljs_devtools.clj (hunk 3 above)
Comment (pepe) [reply src/powerlaces/boot_cljs_devtools.clj]: Well that's what my Spacemacs call: clojure  align. I've seen in many projects, so I am using it. But no problem, with reformatting it. I will do today.

File: README.org (hunk 1 above)
Comment (pepe) [reply README.org]: Agreed, just followed current state. The thing is Dirac is one of the fastest moving targets around 😉

File: src/powerlaces/boot_cljs_devtools.clj (hunk 2 above)
Comment (pepe) [reply src/powerlaces/boot_cljs_devtools.clj]: Or maybe keyword param and map from which the right symbol will be pulled?

**Comment by pepe** [comment]
If you can wait until tomorrow, I will do the things from above and come back 😉

File: src/powerlaces/boot_cljs_devtools.clj (hunk 2 above)
Comment (pepe) [reply src/powerlaces/boot_cljs_devtools.clj]: Well, this one I will think through little bit more 😉

**Comment by arichiardi** [comment]
Yep I can wait no problem, thanks for taking a second look. The comments are minor, but I really appreciate the effort 😀
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by arichiardi** [comment]
@pepe great thanks! I am going to squash and merge this!

**Pull request merged**